app/src/main/java/framgia/com/ichat/screen/privateroom/PrivateRoomFragment.java

```diff
@@ -35,6 +37,9 @@ protected void initComponents() {
         RecyclerView recyclerView = getView().findViewById(R.id.recycler_private_room);
         mPrivateRooms = new ArrayList<>();
         getView().findViewById(R.id.fab_private_room).setOnClickListener(this);
+        mAdapter = new PrivateRoomAdapter(getActivity(), mPrivateRooms, LayoutInflater.from(getActivity()));
+        recyclerView.setAdapter(mAdapter);
+        recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
```

Comment:
cái này a chỉ suggest thôi nhé. 
e có thể set layoutManager bằng cách này hoặc set trực tiếp trong xml (tùy trường hợp)
```suggestion
 recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
```
or
```suggestion
<android.support.v7.widget.RecyclerView
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:orientation="vertical"
                app:layoutManager="android.support.v7.widget.LinearLayoutManager"
                />
```